osquery/tables/system/posix/sudoers.cpp

```diff
@@ -61,6 +62,20 @@ void genSudoersFile(const std::string& filename,
       continue;
     }
 
+    // if last line contains a backslash as the last character,
+    // treat current line as part of previous line and
+    // append it to appropriate column.
+    if (isLongLine) {
```

Comment:
Nit: if following the convention of the rest of this file, variable names are in `snake_case` rather than `camelCase`

Reply:
changed variable name to snake_case instead of caseCase.